doc/classes/TileSetAtlasSource.xml

```diff
@@ -20,16 +20,16 @@
 			<param index="0" name="atlas_coords" type="Vector2i" />
 			<param index="1" name="alternative_id_override" type="int" default="-1" />
 			<description>
-				Creates an alternative tile for the tile at coords [code]atlas_coords[/code]. If [code]alternative_id_override[/code] is -1, give it an automatically generated unique ID, or assigns it the given ID otherwise.
-				Returns the new alternative identifier, or -1 if the alternative could not be created with a provided [code]alternative_id_override[/code].
+				Creates an alternative tile for the tile at coords [param atlas_coords]. If [param alternative_id_override] is -1, give it an automatically generated unique ID, or assigns it the given ID otherwise.
```

Comment:
```suggestion
				Creates an alternative tile for the tile at coordinates [param atlas_coords]. If [param alternative_id_override] is -1, give it an automatically generated unique ID, or assigns it the given ID otherwise.
```